hicommonwealth/commonwealth
# 6786 number input bug

## Link to Issue
Closes: #6786 

## Description of Changes
- Adds a check to `quill_renderer` for the delta JSON format since our switch to markdown 

## "How We Fixed It"

- Implemented a check for bracket to ensure it's in the delta format, resolves the incorrect 'unknown' format for a number only input

## Test Plan
- Go to a thread or create a thread
- Input numbers only
- Ensure it submits correctly
